src/main/webapp/styles/recommendation-map.css

```diff
@@ -19,6 +23,15 @@ h3, p, label {
   width: 18%;
 }
 
+/* Category Button Colours */
```

Comment:
![image](https://user-images.githubusercontent.com/41379082/93594286-77a23900-f9ad-11ea-90bb-222a6d5fb3b6.png)

Reply:
I think, rather than green, we should use the grey of the initial marker placed? It's just a small thing but just to reduce the amount of colours on the screen.